src/PinchZoom/component.tsx

```diff
@@ -928,14 +929,26 @@ class PinchZoom extends React.Component<RequiredProps & DefaultProps> {
         ["touchmove", this._handlerOnTouchMove]
       ]
     : [
-        ["mousemove", this.simulate(this._handlerOnTouchMove), document],
-        ["mouseup", this.simulate(this._handlerOnTouchEnd), document],
+        [
+          "mousemove",
+          this.simulate(this._handlerOnTouchMove),
+          this.state.document
+        ],
+        [
+          "mouseup",
+          this.simulate(this._handlerOnTouchEnd),
+          this.state.document
+        ],
         ["mousedown", this.simulate(this._handlerOnTouchStart)],
         ["click", this._handleClick],
         ["wheel", this._handlerWheel]
       ];
 
   componentDidMount() {
+    const document = window.document;
+    const { documentElement: _html, body: _body } = document;
+
+    this.setState({ _html, _body, document });
```

Comment:
How about don't invoke a browser-specific code, and keep a component state less?

```js
if ( !isSSR ) {
  this._bindEvents();
  this._update();	    
}
```